### PR TITLE
Add Fetkovich IPR Case Description

### PR DESCRIPTION
**Pull request by caio-ramos**
ASIM-5245

Add a `CaseDescription` of the Fetkovich IPR model.

### CODE REVIEW

**Review by nicoddemus — APPROVED** [review]

File: src/alfasim_sdk/_internal/alfacase/case_description.py [hunk 1]
```diff
@@ -1144,6 +1144,33 @@ class VogelIPRDescription(CommonIPR):
     # [[[end]]] (checksum: 9934660f6467a25e11bfbe4c95f8faa7)
 
 
+@attr.s(frozen=True, slots=True)
+class UndersaturatedOilIPRDescription(CommonIPR):
```
Comment (ggrbill) [inline src/alfasim_sdk/_internal/alfacase/case_description.py:1148]: As I suggested in the alfasim PR. Please, consider renaming this class to `FetkovichIPRModel`.

**Review by ggrbill — APPROVED** [review]

File: tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py [hunk 2]
```diff
@@ -359,6 +359,13 @@
     well_max_flow_rate=Scalar(1.0, "sm3/d"),
     well_max_flow_rate_curve=Curve(Array([1.0, 1.0], "sm3/d"), Array([0, 1], "s")),
 )
+UNDERSATURATED_OIL_IPR_DESCRIPTION = case_description.UndersaturatedOilIPRDescription(
+    well_index_phase=constants.WellIndexPhaseType.Oil,
+    min_pressure_difference=Scalar(0.0, "bar"),
+    bubble_point_pressure=Scalar(0.0, "bar"),
+    well_index=Scalar(1.0e-6, "m3/Pa.s"),
+    well_index_curve=Curve(Array([1.0e-6, 0.9e-6], "m3/Pa.s"), Array([0, 1], "h")),
+)
```
Comment (prusse-martin) [inline tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py:368]: Those should be values diferent from the default values to check if the conversion "case description"<->"alfacase" round trip is no losing any data.

**Review by prusse-martin — CHANGES_REQUESTED** [review]

File: tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py (hunk 2 above)
Comment (eniohayashi) [reply tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py:368]: Looks like Vogel values are the default ones too, right? If so, they could be fixed here.

File: src/alfasim_sdk/_internal/alfacase/case_description.py (hunk 1 above)
Comment (caio-ramos) [reply src/alfasim_sdk/_internal/alfacase/case_description.py]: Done

File: tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py (hunk 2 above)
Comment (caio-ramos) [reply tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py]: Thanks, both are done!

**Review by eniohayashi — APPROVED** [review]

**Review by prusse-martin — APPROVED** [review]

**Pull request merged**